### PR TITLE
Add colors to previously uncolored elements

### PR DESCRIPTION
**Pull request by PythonTryHard**
Add colors to `async`, `classmethod`, `instance-attribute`, `class-attribute`, and `property`. Color choice is arbitrary, and I'm not sure I caught everything.

Color can be changed by modifying the variables at the top following [this StackOverflow question and answer](https://stackoverflow.com/questions/12216572/add-alpha-channel-to-a-given-color). Everything needed should be in the comments.

### CODE REVIEW

**Comment by PythonTryHard** [comment]
```
/*  For some godawful reasons, "classmethod" is rendered as:
 *      <div>
 *          <code>
 *             <mark>classmeth</mark>
 *              od
 *          </code>
 *      </div>
 */
```
This is not consistent. On my machine, it's `classmeth` + `od`; while the [live docs](https://arc.hypergonial.com/api_reference/abc/option/?h=classmethod#arc.abc.option.OptionType.from_hikari) is `classmethod`. The only consistent thing is the `<mark>`

File: docs/stylesheets/additional_coloring.css [hunk 1]
```diff
@@ -0,0 +1,58 @@
+[data-md-color-primary="teal"] {
+    --async-color: 255, 92, 122;
+    --property-color: 38, 166, 153;
+    --classmethod-color: 0, 198, 245;
+    --class-attribute-color: 230, 170, 0;
+    --instance-attribute-color: 255, 61, 220;
```
Comment (hypergonial) [inline docs/stylesheets/additional_coloring.css]:
```suggestion
    --instance-attribute-color: 181, 156, 255;
```

How about a color that is a little easier on the eyes for this one? Looks something like this:

Screenshot_20240504_215107

**Review by hypergonial — CHANGES_REQUESTED** [review]
Sorry for taking so long to get back to you on this, just one minor nitpick :)

File: docs/stylesheets/additional_coloring.css (hunk 1 above)
Comment (PythonTryHard) [reply docs/stylesheets/additional_coloring.css]: No problemo

**Review by hypergonial — APPROVED** [review]

**Pull request merged**